src/main/java/be/syntra/devshop/DevshopBack/entities/Address.java

```diff
@@ -31,7 +31,7 @@
     @NotNull
```

Comment:
@NotBlank already incorporates the @NotNull. Remove unnecessary NotNull annotations

Reply:
fixed